user_events.py

```diff
@@ -409,9 +417,13 @@ def user_an_event_edit(id):
                     # unlink between subevent and super event.
                     if 'id' in old_sub_event:
                         update_super_event_id(old_sub_event['id'], '')
```

Comment:
This is unreleased to this PR, but I think it's better to rename this method as `update_super_event_by_platform_id`. Thanks.